src/NzbDrone.Core/Indexers/Definitions/RuTracker.cs

```diff
@@ -0,0 +1,1809 @@
+            var forum = row.QuerySelector("td.f-name-col > div.f-name > a");
+            var forumid = forum.GetAttribute("href").Split('=')[1];
+            return _categories.MapTrackerCatToNewznab(forumid);
+        }
+
+        private long GetSizeOfRelease(in IElement row)
+        {
+            var qSize = row.QuerySelector("td.tor-size");
+            var size = ReleaseInfo.GetBytes(qSize.GetAttribute("data-ts_text"));
+            return size;
+        }
+
+        private DateTime GetPublishDateOfRelease(in IElement row)
+        {
+            var timestr = row.QuerySelector("td:nth-child(10)").GetAttribute("data-ts_text");
+            var publishDate = DateTimeUtil.UnixTimestampToDateTime(long.Parse(timestr));
+            return publishDate;
+        }
+
+        private bool IsAnyTvCategory(ICollection<IndexerCategory> category)
+        {
+            return category.Contains(NewznabStandardCategory.TV)
+                || NewznabStandardCategory.TV.SubCategories.Any(subCat => category.Contains(subCat));
+        }
+
+        private bool IsAnyMovieCategory(ICollection<IndexerCategory> category)
+        {
+            return category.Contains(NewznabStandardCategory.Movies)
+                || NewznabStandardCategory.Movies.SubCategories.Any(subCat => category.Contains(subCat));
+        }
+
+        public Action<IDictionary<string, string>, DateTime?> CookiesUpdater { get; set; }
+    }
+
+    public class RuTrackerSettingsValidator : AbstractValidator<RuTrackerSettings>
+    {
+        public RuTrackerSettingsValidator()
+        {
+            RuleFor(c => c.Username).NotEmpty();
+            RuleFor(c => c.Password).NotEmpty();
+        }
+    }
+
+    public class RuTrackerCaptcha
+    {
+        public string ContentType { get; set; }
+        public byte[] ImageData { get; set; }
+        public string CaptchaField { get; set; }
+        public string CaptchaSid { get; set; }
+    }
+
+    public class RuTrackerSettings : IIndexerSettings
+    {
+        private static readonly RuTrackerSettingsValidator Validator = new RuTrackerSettingsValidator();
+
+        public RuTrackerSettings()
+        {
+            Username = "";
+            Password = "";
+            RussianLetters = false;
+        }
+
+        [FieldDefinition(1, Label = "Base Url", Type = FieldType.Select, SelectOptionsProviderAction = "getUrls", HelpText = "Select which baseurl Prowlarr will use for requests to the site")]
+        public string BaseUrl { get; set; }
+
+        [FieldDefinition(2, Label = "Username", Advanced = false, HelpText = "Site Username")]
+        public string Username { get; set; }
+
+        [FieldDefinition(3, Label = "Password", Type = FieldType.Password, Privacy = PrivacyLevel.Password, HelpText = "Site Password")]
+        public string Password { get; set; }
+
+        [FieldDefinition(4, Label = "Captcha Text", Type = FieldType.Captcha, SelectOptionsProviderAction = "checkCaptcha", HelpText = "Captcha Text")]
```

Comment:
Should we just rip out this Captcha stuff given it's not really doing anything here?

Reply:
Tore out everything captcha from the code. Tested that everything still works.